### DISCUSSION TITLE
如何避免别人挂载我的alist

### DISCUSSION THREAD

**Discussion by tmzg0000** [AlistGo/alist#3615] [General]
我开了访客访问，别人知道了网址就能挂载我的alist吗？ 
我搜索了文档没找到答案。
我看网上有些其他人的alist知道网站也不能挂载，挂载后是空的。是怎么做到的？

**Comment by Bssn520** [comment]
后台管理--->设置--->站点--->允许索引，关了应该就可以了。

**Reply by tmzg0000** [reply]
我默认就是关闭的，确认不是这个地方关了的问题

**Comment by anwen-anyi** [comment]
挂载别人是空的，那是他加密了 不知道密码挂上去就会这样
下面的哪个老哥说的是可以挂载能看到文件，不能索引
加密后别人挂载你的站点需要密码才符合你的需求

**Reply by tmzg0000** [reply]
请问如何加密，具体设置路径在什么地方

**Reply by anwen-anyi** [reply]
元信息加密，如果你个别目录想公开就单独加个别文件
如果都不想公开你就直接加密整个根目录 不过这样游客也就不能访问了

**Comment by lishuda** [comment]
关闭 访客账户可以做到，只有知道你账号密码的人可以查看挂载

**Reply by tmzg0000** [reply]
我有部分内容想让人看到，只是不想被其他alist挂载而已。你这样关闭访客就所有都看不到了

**Reply by lishuda** [reply]
设置里有”元信息“一项，这一项可以为目录添加密码，只有登陆或者知道密码的访客才可以访问，应该可以满足你的要求

**Comment by wyhwangyuhengg** [comment]
我有办法。访客是有个用户的，访客用户不开启    Webdav 读取    这个功能应该就行了。我猜的。。。

**Reply by tmzg0000** [reply]
访客默认就是关闭的啊

**Reply by wyhwangyuhengg** [reply]
那就应该不能挂载吧，要不你把你的alist给我，我来试试

**Comment by Nurbury** [comment]
试试设置Guest用户的基本路径。将想对外显示的内容放入该目录中，这样别人挂载你的alist后，他以游客身份登录，只能看到这个目录放置的文件。

**Reply by tmzg0000** [reply]
我是完全不想给别人挂，但是有内容想给guest看。你说的方法给guest的就能被别人挂上

**Comment by HCARX** [comment]
同问，不然非本站访问量多了就出现问题

**Comment by And8595A** [comment]
游客访问怎么开啊
